src/components/Portfolio/Portfolio.tsx

```diff
@@ -90,9 +93,9 @@ const Portfolio: React.FC = () => {
               </Text>
               {renderTotalBalance()}
             </Box>
-            <Box width="100%" height="100%" minHeight="80px">
+            <Box width="100%" height="100%" minHeight="140px">
```

Comment:
somehow alignment is not proper 

![Screenshot 2022-07-08 at 10 28 26 AM](https://user-images.githubusercontent.com/6604146/177920275-9caf4b98-399b-42df-9481-e1ee2960e516.png)